app/views/RequiredInformationView.scala.html

```diff
@@ -18,40 +18,42 @@
 
 @this(
     layout: templates.Layout,
-    formHelper: FormWithCSRF,
     govukErrorSummary: GovukErrorSummary,
-    govukCheckboxes: GovukCheckboxes,
     caption: Caption,
     heading: Heading,
     paragraph: Paragraph,
-    saveButtons: SaveButtons
+    saveButtons: SaveButtons,
+    insetPara: InsetPara,
+    bulletList: BulletList
 )
 
-@(form: Form[_], authUserType: AuthUserType, draftId: DraftId)(implicit request: Request[_], messages: Messages)
+@(draftId: DraftId)(implicit request: Request[_], messages: Messages)
 
-@paraContent = {@paragraph(Html(messages("requiredInformation.hint")))}
+@layout(pageTitle = titleNoForm(messages("requiredInformation.title")), draftId = Some(draftId)) {
 
-@layout(pageTitle = title(form, messages("requiredInformation.title")), draftId = Some(draftId)) {
+        @caption(messages("requiredInformation.caption"))
 
-    @formHelper(action = routes.RequiredInformationController.onSubmit(draftId), Symbol("autoComplete") -> "off") {
+        @heading(messages("requiredInformation.heading"))
 
-        
-        @if(form.errors.nonEmpty) {
-            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
-        }
-        
-        @caption(messages("requiredInformation.caption"))
+        @insetPara(Html(messages("requiredInformation.inset.para")))
 
-        @govukCheckboxes(
-            CheckboxesViewModel(
-                form   = form,
-                name   = "value",
-                legend = LegendViewModel(messages("requiredInformation.heading")).asPageHeading(),
-                items  = RequiredInformation.checkboxItems(authUserType)
+        @bulletList(
+            Seq(
+                Html(messages("requiredInformation.option1.individual")),
+                Html(messages("requiredInformation.option2.individual")),
+                Html(messages("requiredInformation.option3.individual")),
+                Html(messages("requiredInformation.option4.individual")),
+                Html(messages("requiredInformation.option5.individual")),
+                Html(messages("requiredInformation.option6.individual"))
             )
-            .withHint(Hint(content = HtmlContent(paraContent)))
         )
 
-        @saveButtons()
-    }
+        <span class="govuk-button-group">
+                <a href="@routes.ImportGoodsController.onPageLoad(NormalMode, draftId).url" role="button" class="govuk-button" id="continue-button">
+                    @messages("site.saveAndContinue")
+                </a>
+                <a href="@routes.AccountHomeController.onPageLoad().url" role="button" class="govuk-button govuk-button--secondary">
+                    @messages("site.saveAsDraft")
+                </a>
```

Comment:
I wonder if it's sensible to remove 'save as draft' on this page (and the other version) as there's nothing to save